### PR TITLE
Aligned Generator.cpp with actual Dumper (FUObjectItem now contains C…

### PR DESCRIPTION
**Pull request by Uhrwerk7**
…lusterIndex); Added ProcessEvent to UObject

### CODE REVIEW

File: Target/PUBG/Generator.cpp [hunk 1]
```diff
@@ -145,6 +145,10 @@ class Generator : public IGenerator
 		}
 		return nullptr;
 	})"),
+			PredefinedMethod::Inline(R"(	inline void ProcessEvent(class UFunction* function, void* parms)
```
Comment (KN4CK3R) [inline Target/PUBG/Generator.cpp]: The pattern from here doesn't work anymore? https://github.com/Uhrwerk7/UnrealEngineSDKGenerator/blob/e99e8379f19eb672ebab705f6e6f7f5dda78c93b/Target/PUBG/Generator.cpp#L20

Comment (Uhrwerk7) [reply Target/PUBG/Generator.cpp]: UObject doesn't have Process Event in the Header anymore which leads to Compiler Errors without this Change.

Comment (KN4CK3R) [reply Target/PUBG/Generator.cpp]: Sure, but the correct way is to fix the signature instead of adding a new method.

Comment (Uhrwerk7) [reply Target/PUBG/Generator.cpp]: I will look into finding the correct Signature during the course of the Weekend, since i'm going to a Fair from today till Saturday.

Comment (KN4CK3R) [reply Target/PUBG/Generator.cpp]: Could you revert the ProcessEvent change? I will merge this PR then and you can create a new PR with the ProcessEvent changes when you are back.

Comment (Uhrwerk7) [reply Target/PUBG/Generator.cpp]: Give me an Hour, Will do.

Comment (Uhrwerk7) [reply Target/PUBG/Generator.cpp]: You should be able to merge with the new Changes now.

File: Engine/Package.cpp [hunk 2]
```diff
@@ -118,8 +118,8 @@ void Package::GeneratePrerequisites(const UEObject& obj, std::unordered_map<UEOb
 	}
 
 	const auto name = obj.GetName();
-	if (name.find("Default__") != std::string::npos
-		|| name.find("<uninitialized>") != std::string::npos
+	if (/*name.find("Default__") != std::string::npos
```
Comment (KN4CK3R) [inline Engine/Package.cpp]: Why did you remove this?

Comment (Uhrwerk7) [reply Engine/Package.cpp]: I was about to revert the commit again (by mistake).
